typescript/infra/config/environments/testnet4/chains.ts

```diff
@@ -18,11 +18,6 @@ export const chainMetadataOverrides: ChainMap<Partial<ChainMetadata>> = {
       gasPrice: 8 * 10 ** 9, // 8 gwei
     },
   },
-  scrollsepolia: {
```

Comment:
introduced in https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/4297, have to remove to get any txs through (base fee was higher than this forced price)